app/services/allocation_service.rb

```diff
@@ -69,10 +69,12 @@ def self.create_or_update(params)
     alloc_version
   end
 
-  def self.active_allocations(nomis_offender_ids, prison)
-    Allocation.active(nomis_offender_ids, prison).index_by { |a|
-      a[:nomis_offender_id]
-    }
+  def self.previously_allocated_poms(nomis_offender_id)
+    allocation = Allocation.find_by(nomis_offender_id: nomis_offender_id)
+
+    return [] if allocation.nil?
+
+    allocation.get_old_versions.map(&:primary_pom_nomis_id)
```

Comment:
Looks like this method might duplicate `Allocation#previously_allocated_poms`.

Is it needed? I can't find any uses of this method anywhere, aside from in its own spec file.

Reply:
Thought I had got rid of this - it seems to have come back